jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/DataTableAssert.java

```diff
@@ -27,154 +27,179 @@
  */
 public class DataTableAssert<L extends Section, D>
         extends BaseTableAssert<DataTable<L, D>, DataTableAssert<L, D>> {
-
+    
     /**
      * Check that the table has rows that meet expected condition
+     *
      * @param condition to compare
      * @return DataTableAssert
      */
     @JDIAction("Assert that '{name}' has rows that meet expected condition")
-    public DataTableAssert<L, D> row(JFunc1<D,Boolean> condition) {
-        jdiAssert(table().dataRow(condition), not(nullValue()));
+    public DataTableAssert<L, D> row(JFunc1<D, Boolean> condition, String... messages) {
+        jdiAssert(table().dataRow(condition), not(nullValue()), messages);
         return this;
     }
+    
     @JDIAction("Assert that '{name}' has rows that meet expected condition")
-    public DataTableAssert<L, D> value(JFunc1<D,Boolean> condition, Row row) {
-        jdiAssert(condition.execute(table().dataRow(row.getIndex(table().header()))), Matchers.is(true));
+    public DataTableAssert<L, D> value(JFunc1<D, Boolean> condition, Row row, String... messages) {
+        jdiAssert(condition.execute(table().dataRow(row.getIndex(table().header()))), Matchers.is(true), messages);
         return this;
     }
+    
     /**
      * Check that the table has row that has expected data
+     *
      * @param data to compare
      * @return DataTableAssert
      */
     @JDIAction("Assert that '{name}' has {0}")
-    public DataTableAssert<L, D> row(D data) {
-        return row(d -> d.equals(data));
+    public DataTableAssert<L, D> row(D data, String ... messages) {
+        return row(d -> d.equals(data), messages);
     }
+    
     @JDIAction("Assert that '{name}' has {0}")
-    public DataTableAssert<L, D> value(D expected, Row row) {
+    public DataTableAssert<L, D> value(D expected, Row row, String ... messages) {
         D actual = table().dataRow(row.getIndex(table().header()));
-        jdiAssert(actual, Matchers.is(expected));
+        jdiAssert(actual, Matchers.is(expected), messages);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)